addons/pos_restaurant/static/src/overrides/components/order_widget/order_widget.js

```diff
@@ -0,0 +1,18 @@
+/** @odoo-module */
+
+import { OrderWidget } from "@point_of_sale/app/generic_components/order_widget/order_widget";
+import { patch } from "@web/core/utils/patch";
+import { _t } from "@web/core/l10n/translation";
+
+OrderWidget.props.isConfigRestaurant = { type: Boolean, optional: true };
+OrderWidget.props.isOrderBooked = { type: Boolean, optional: true };
```

Comment:
```suggestion
patch(OrderWidget.props, {
    ...OrderWidget.props, ...{
        isConfigRestaurant: { type: Boolean, optional: true },
        isOrderBooked: { type: Boolean, optional: true },
    }
});
```

I'm not sure if that's the exact code, but the idea is to use patch.